packages/backend/src/panels/YeomanUIPanel.ts

```diff
@@ -158,5 +159,6 @@ export class YeomanUIPanel extends AbstractWebviewPanel {
   public initWebviewPanel() {
     super.initWebviewPanel();
     this.webViewPanel.title = this.messages.panel_title;
+    void notifyGeneratorsInstallationProgress(this, this.webViewPanel);
```

Comment:
do you need 2 args ? `webViewPanel` is the member of YeomanUIPanel `this` instance.

Reply:
`webViewPanel` is protected, should the code use it in a way like this: `panel['webViewPanel'] ?